### PR TITLE
[ru] cs-229-unsupervised-learning

### PR DESCRIPTION
**Pull request by AlexandrParkhomenko**
web format: [Unsupervised Learning](https://alexandrparkhomenko.github.io/stanford/cs-229/unsupervised-learning/index.html)

### CODE REVIEW

File: ru/cs-229-unsupervised-learning.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+**8. [Setting, Latent variable z, Comments]**
+
+&#10230; [Настройка, Скрытая величина z, Комментарии]
+
+<br>
+
+**9. [Mixture of k Gaussians, Factor analysis]**
+
+&#10230; [Смесь k гауссианов, Факторный анализ]
```
Comment (geotrush) [inline ru/cs-229-unsupervised-learning.md:55]:
```suggestion
&#10230; [Смесь k гауссиан, Факторный анализ]
```

File: ru/cs-229-unsupervised-learning.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+**50. Therefore, the stochastic gradient ascent learning rule is such that for each training example x(i), we update W as follows:**
+
+&#10230; Следовательно, правило обучения стохастическому градиентному восхождению таково, что для каждого обучающего примера x(i) мы обновляем W следующим образом:
+
+<br>
+
+**51. The Machine Learning cheatsheets are now available in [target language].**
+
+&#10230; Шпаргалки по машинному обучению теперь доступны в формате [target language].
```
Comment (geotrush) [inline ru/cs-229-unsupervised-learning.md:307]:
```suggestion
&#10230; Шпаргалки по машинному обучению теперь доступны на русском языке.
```

File: ru/cs-229-unsupervised-learning.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+<br>
+
+**52. Original authors**
+
+&#10230; Авторы оригинала: Afshine Amidi и Shervine Amidi ― https://github.com/shervinea
```
Comment (geotrush) [inline ru/cs-229-unsupervised-learning.md:313]:
```suggestion
&#10230; Авторы оригинала: Afshine Amidi и Shervine Amidi ― https://github.com/afshinea и https://github.com/shervinea
```

File: ru/cs-229-unsupervised-learning.md [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+
+<br>
+
+**53. Translated by X, Y and Z**
+
+&#10230; Российская адаптация: Пархоменко Александр ― https://github.com/AlexandrParkhomenko
```
Comment (geotrush) [inline ru/cs-229-unsupervised-learning.md:319]:
```suggestion
&#10230; Переведено на русский язык: Пархоменко Александр ― https://github.com/AlexandrParkhomenko
```

File: ru/cs-229-unsupervised-learning.md [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+<br>
+
+**54. Reviewed by X, Y and Z**
+
+&#10230; Проверено X, Y и Z
```
Comment (geotrush) [inline ru/cs-229-unsupervised-learning.md:325]:
```suggestion
&#10230; Проверено на русском языке: Труш Георгий (Georgy Trush) ― https://github.com/geotrush
```

**Review by tensorush — CHANGES_REQUESTED** [review]
Amazing! I've made minor corrections.

**Pull request closed without merging**